test/integration/custom-routes/test/index.test.js

```diff
@@ -962,7 +962,7 @@ const runTests = (isDev = false) => {
       host: '1',
     })
 
-    const res2 = await fetchViaHTTP(appPort, '/has-rewrite-3')
+    const res2 = await fetchViaHTTP(appPort, '/has-rewrite-4')
```

Comment:
Why was this changed?

Reply:
Since this test is for `/has-rewrite-4` but I noticed there is another check for 404 so maybe we can just delete this.